spec/syntax.md

```diff
@@ -97,7 +97,7 @@ Attempting to parse a _message_ that is not _valid_ will result in a _Data Model
 
 A **_<dfn>message</dfn>_** is the complete template for a specific message formatting request.
 
-A **_<dfn>variable</dfn>_** is a _name_ associated to a resolved value.
+A **_<dfn>variable</dfn>_** is a _name_ associated with a _resolved value_.
```

Comment:
I think the original was better? Associated to is specific.

Reply:
I find this all somewhat muddled. Except in function composition, "resolved value" is not a useful abstraction. 

That is, outside of function composition, the relevant features are 
- a formatted value, and
- a selection value

Reply:
```suggestion
A **_<dfn>variable</dfn>_** is a _name_ associated to a _resolved value_.
```